config/routes.rb

```diff
@@ -3,4 +3,6 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "guests/:id", to: "guests#show"
+  post "guests/:id", to: "guest_rooms#create"
```

Comment:
since this is going to the `guest_rooms` controller, which I agree with doing, it would make a bit more sense to use "guest_rooms" in your URL.

Reply:
Or, you could do something like /guests/:id/rooms

Reply:
do you mean like `post "guest_rooms/:id", to: "guest_rooms#create"` ? I guess what confuses me there is that I was working on the guest show page so I thought the route needed to be the same url.

If I did `/guests/:id/rooms` what else would I need to do, I know I would need to change the form in my show page to reflect that url, anything else?

Reply:
The route on your form directs that form submission to an action that creates a new GuestRoom resource, and then that action redirects to the Guest show page.. so, they don't need to look the same. Your action will redirect where you need it to go.

If you did `/guests/:id/rooms`, you're right, you'd need to change the form and your routes file, but that should be it!